spec/requests/api/tenant_quotas_spec.rb

```diff
@@ -0,0 +1,246 @@
+describe "tenant quotas API" do
+  let(:tenant) { FactoryGirl.create(:tenant) }
```

Comment:
@imtayadeway I tried to "abstract" all the `quota_1 = ...` lines as the gist link shows, but I have some weird errors, like tenant and quotas are created, but the api returns no quota ... So I don't know if it is a bug on our side, or something funky in this spec :( https://gist.github.com/romaintb/34a6f9b62776a72a5f4605bee328903a